src/ppx/grfx/vk/vk_command.cpp

```diff
@@ -290,36 +290,63 @@ void CommandBuffer::ClearRenderTarget(
     grfx::Image*                        pImage,
     const grfx::RenderTargetClearValue& clearValue)
 {
-    auto pCurrentRenderPass = GetCurrentRenderPass();
-    if (IsNull(pCurrentRenderPass)) {
+    if (!HasActiveRenderPass()) {
```

Comment:
I know it was like this before, but perhaps we should add a warning here, just in case someone expects this to work outside a render pass but it silently does not